File: pset/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void) {
    int height = 0;
    height = get_int("\nIngrese la altura de la piramide\n");


    for(int i = 0; i<=height - 1; i++) {
        for(int j = 0; j <= height; j++) {
            if(j > height - i) {
                printf("#");
            } else {
                printf("-");
            }
            if(j == height) {
                printf("\n");
            }
        }
        printf("\n");
    }

    return 0;
}

File: psets/pset6/sentimental/mario.py
from cs50 import get_int


def main():

    height = 0
    while(True):
        height = get_int('Ingrese la altura de la piramide')
        if(height >= 0 and height <= 23):
            break

    if height == 0:
        return 'EOF'
    width = height + 1

    for i in range(height):
        for j in range(width):
            if j > height - i - 2:
                print('#', end="")
            elif j < width:
                print(' ', end="")
        print('')


main()
